x-pack/plugin/monitoring/src/main/java/org/elasticsearch/xpack/monitoring/MonitoringTemplateRegistry.java

```diff
@@ -78,7 +78,7 @@
      * writes monitoring data in ECS format as of 8.0. These templates define the ECS schema as well as alias fields for the old monitoring
      * mappings that point to the corresponding ECS fields.
      */
-    public static final int STACK_MONITORING_REGISTRY_VERSION = Version.V_8_0_0.id;
+    public static final int STACK_MONITORING_REGISTRY_VERSION = Version.V_8_2_0.id;
```

Comment:
Need to be careful and update the backported PRs to assign the correct version

Reply:
Discussed on slack, but keeping here for visibility:

Hmm, we should be careful here. This is used to determine if we need to upgrade templates that are present. We probably shouldn't have set this to the product version, but that was the historical approach for monitoring indices that was already in place, so we have been somewhat locked in.

Check out the StackTemplateRegistry file in the stack plugin. That has a version of `1` right now. This version number needs to monotonically increase every update to the templates, and the version number should be the same for identical mappings.

```suggestion
    public static final int STACK_MONITORING_REGISTRY_VERSION = Version.V_8_0_0.id + 1;
```